tools/biobb_pytorch/biobb_apply_mdae.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="biobb_pytorch_apply_mdae" name="ApplyMdae" version="4.2.1" >
+    <description>Apply a Molecular Dynamics AutoEncoder (MDAE) PyTorch model.</description>
+
+    <requirements>
+      <requirement type="package" version="4.2.1">biobb_pytorch</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+
+      ln -s $input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext;
```

Comment:
```suggestion
      ln -s $input_data_npy_path ./input_data_npy_path.$input_data_npy_path.ext &&
```

I think this is preferable (and on following lines), so the script stops executing on the first failure